### PR TITLE
Get python console online in Qt6 builds

### PR DESCRIPTION
**Pull request by nyalldawson**
Some fixes necessary to get python initialized on qt 6 builds and get the python console functional

### CODE REVIEW

File: scripts/pyqt5_to_pyqt6/pyqt5_to_pyqt6.py [hunk 1]
```diff
@@ -173,6 +192,10 @@ def get_class_enums(item):
         if inspect.isclass(value) and type(value).__name__ == 'EnumType':
             for e in value:
                 qt_enums[(item.__name__, e.name)] = f"{value.__name__}"
+
+            for _extra_value in extra_members.get((item.__name__, value.__name__), []):
+                qt_enums[(item.__name__, _extra_value)] = f"{value.__name__}"
+
```
Comment (troopa81) [inline scripts/pyqt5_to_pyqt6/pyqt5_to_pyqt6.py:198]:
```suggestion
for ekey, evalue in value.__dict__.items():
+                if (isinstance(evalue, value)):
+                    qt_enums[(item.__name__, ekey)] = f"{value.__name__}"
```

This should solve things for all extra_members like enum

Comment (nyalldawson) [reply scripts/pyqt5_to_pyqt6/pyqt5_to_pyqt6.py:198]: Thanks, that's indeed much better!

File: python/utils.py [hunk 2]
```diff
@@ -884,6 +884,15 @@ def _import(name, globals={}, locals={}, fromlist=[], level=None):
         msg = 'PyQt4 classes cannot be imported in QGIS 3.x.\n' \
               'Use {} or the version independent {} import instead.'.format(name.replace('PyQt4', 'PyQt5'), name.replace('PyQt4', 'qgis.PyQt'))
         raise ImportError(msg)
+    qt_version = int(QT_VERSION_STR.split('.')[0])
+    if qt_version == 5 and 'PyQt6' in name:
+        msg = 'PyQt6 classes cannot be imported in a QGIS build based on Qt5.\n' \
+              'Use {} or the version independent {} import instead.'.format(name.replace('PyQt6', 'PyQt5'), name.replace('PyQt6', 'qgis.PyQt'))
```
[the remark this answers is not in the document]
Comment (troopa81) [reply python/utils.py:890]: Right. Can I suggest a message a little more incentive like

> Use {} or preferably the version independent {} import instead when available.

**Review by troopa81 — APPROVED** [review]
One minor comment for a more incentive message, the rest looks good

File: python/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nyalldawson) [reply python/utils.py:890]: Ok, done!

**Pull request merged**